### DISCUSSION TITLE
'Plans for MSSQL supported db'

### DISCUSSION THREAD

**Discussion by lisaconnect1234** [prisma/prisma#8256] [Q&A] [answered]
## Problem

I see MSSQL is in preview feature state, what are your plans for full support of MSSQL?

**Comment by tomhoule** [comment, marked as answer]
This is still being actively worked on, and getting closer to release. We don't have a fixed date for the stable release yet, however.

**Comment by lisaconnect1234** [comment]
> This is still being actively worked on, and getting closer to release. We don't have a fixed date for the stable release yet, however.

This is on the roadmap and the plan is to support MSSQL, correct?  Can you check on potential release dates?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
